### PR TITLE
Updated December Release Notes

### PR DESCRIPTION
**Pull request by biodranik**
A follow-up for https://github.com/organicmaps/organicmaps/pull/6881

### CODE REVIEW

File: iphone/metadata/en-US/release_notes.txt [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
-• Added Hackerspace
-• Fixed leisure garden areas
-• Consistent displaying of house numbers on zoom 17
+
+Styles
+• Singapore metro icon
+• Hackerspace
+• Fix Porto metro icon
+• Fixed residential leisure garden area fills disappearing on some zoom levels
+• Display house numbers regardless of buildings' sizes
```
Comment (pastk) [inline iphone/metadata/en-US/release_notes.txt:26]: "on z17"
on z16 the building size logic is used still
though I understand that this "zXX" notion is not clear to the users (and maybe the whole concept of descreet zoom levels)...

File: iphone/metadata/en-US/release_notes.txt [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
 
 Other notable fixes and improvements:
 • Fixed several crashes and freezing on iOS 12, 13, 14
 • Multiple KML files can be imported from a single KMZ file
-• A credit card is displayed for places with ATM
-• A dollar emoji is displayed where a fee is requred
+• Less "gaps" in highways on the World map
+• Removed very short ferry lines from the World map
+• A credit card is displayed for selected places with ATM
+• A dollar emoji is displayed where a fee is required
```
Comment (pastk) [inline iphone/metadata/en-US/release_notes.txt:15]:
```suggestion
• A credit card is displayed in Place Page for POIs with ATM
• A dollar emoji is displayed in Place Page if a fee is required
```

File: iphone/metadata/en-US/release_notes.txt [hunk 3]
```diff
@@ -1,27 +1,34 @@
...
+• CarPlay Search button now properly shows search results
 • Improved user interface for the Search on iPad
-• Added Singapore metro icon and fixed Porto metro icon
-• Added Hackerspace
-• Fixed leisure garden areas
-• Consistent displaying of house numbers on zoom 17
+
+Styles
+• Singapore metro icon
+• Hackerspace
```
Comment (pastk) [inline iphone/metadata/en-US/release_notes.txt:23]:
```suggestion
• Added Hackerspace amenity
```

File: iphone/metadata/en-US/release_notes.txt [hunk 4]
```diff
@@ -1,27 +1,34 @@
...
-• CarPlay search button now properly shows search results
+• CarPlay Search button now properly shows search results
 • Improved user interface for the Search on iPad
-• Added Singapore metro icon and fixed Porto metro icon
-• Added Hackerspace
-• Fixed leisure garden areas
-• Consistent displaying of house numbers on zoom 17
+
+Styles
+• Singapore metro icon
```
Comment (pastk) [inline iphone/metadata/en-US/release_notes.txt:22]:
```suggestion
• Added Singapore metro icon
```

**Review by pastk — APPROVED** [review]

File: iphone/metadata/en-US/release_notes.txt (hunk 2 above)
Comment (biodranik) [reply iphone/metadata/en-US/release_notes.txt:15]: Users are not familiar with the Place Page concept. We use it internally. I've added "selected places" above, to keep the context.

File: iphone/metadata/da/release_notes.txt [hunk 5]
```diff
@@ -1,27 +1,34 @@
...
+• CarPlay Search button now properly shows search results
 • Improved user interface for the Search on iPad
-• Added Singapore metro icon and fixed Porto metro icon
-• Added Hackerspace
-• Fixed leisure garden areas
-• Consistent displaying of house numbers on zoom 17
+
+Styles
+• Singapore metro icon
+• Hackerspace
```
[the remark this answers is not in the document]
Comment (biodranik) [reply iphone/metadata/da/release_notes.txt:23]: https://en.wikipedia.org/wiki/Hackerspace

File: iphone/metadata/da/release_notes.txt [hunk 6]
```diff
@@ -1,27 +1,34 @@
...
 
 Other notable fixes and improvements:
 • Fixed several crashes and freezing on iOS 12, 13, 14
 • Multiple KML files can be imported from a single KMZ file
-• A credit card is displayed for places with ATM
-• A dollar emoji is displayed where a fee is requred
+• Less "gaps" in highways on the World map
+• Removed very short ferry lines from the World map
+• A credit card is displayed for selected places with ATM
+• A dollar emoji is displayed where a fee is required
```
[the remark this answers is not in the document]
Comment (biodranik) [reply iphone/metadata/da/release_notes.txt:15]: Yes, it is worth explaining to users.

File: iphone/metadata/da/release_notes.txt [hunk 7]
```diff
@@ -1,27 +1,34 @@
...
+• Hackerspace
+• Fix Porto metro icon
+• Fixed residential leisure garden area fills disappearing on some zoom levels
+• Display house numbers regardless of buildings' sizes
+• Fine-tuned priorities of many POIs in the main style
 
 Translations
 • Improved Chinese, Dutch, Finnish, Polish, Russian translations
-• Added Dutch, French, Polish, Romanian, and Russian translations for edition Opening Hours instructions
+• Added Dutch, French, Polish, Romanian, and Russian translations for Opening Hours editing instructions
```
[the remark this answers is not in the document]
Comment (biodranik) [reply iphone/metadata/da/release_notes.txt:31]: No. https://dictionary.cambridge.org/dictionary/english/opening-hours

**Pull request merged**